Jenkinsfile

```diff
@@ -92,17 +82,12 @@ pipeline {
   }
 }
 
-def buildImages(){
+def buildAndPushImages(){
+  dockerLogin(secret: "${env.DOCKER_REGISTRY_SECRET}", registry: "${env.REGISTRY}")
+  sh 'docker buildx create --use'
```

Comment:
Otherwise:

```
[2021-01-28T16:35:04.328Z] error: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
```

So I decided to delegate that particular configuration to the CI itself.